app/views/play/CampaignView.coffee

```diff
@@ -449,6 +449,9 @@ module.exports = class CampaignView extends RootView
           unless @levelStatusMap[session.get('levelID')] is 'complete'  # Don't overwrite a complete session with an incomplete one
             @levelStatusMap[session.get('levelID')] = if session.get('state')?.complete then 'complete' else 'started'
           @levelDifficultyMap[session.get('levelID')] = session.get('state').difficulty if session.get('state')?.difficulty
+        if @courseInstance.get('classroomID') == "5d12e7e36eea5a00ac71dc8f"  # Tarena national final classroom
+          unless @levelStatusMap['game-dev-2-final-project']  #make sure all players could access GD2 final on competition day
```

Comment:
Is this supposed to happen on a particular day? If yes then we might need to add that condition, else this will make the level available immediately.

Reply:
If player passed all other levels before the level, it will also be open, so it doesn't matter to let it start earlier. 
btw, they didn't announce this level is the competition level, and Tarena also need to confirm this level open in advance